Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2829.cs

```diff
@@ -143,7 +143,7 @@ public string Text
 			public event PropertyChangedEventHandler PropertyChanged;
 		}
 
-#if UITEST
+#if UITEST && __ANDROID__
 		[Test]
```

Comment:
Doesn't seem to fix the iOS test so much as remove it and gussy up the Android test, eh? Either way...